HdfsAccessor.go

```diff
@@ -118,13 +118,14 @@ func (this *hdfsAccessorImpl) connectToNameNodeImpl(nnAddr string) (*hdfs.Client
 
 // Opens HDFS file for reading
 func (this *hdfsAccessorImpl) OpenRead(path string) (ReadSeekCloser, error) {
```

Comment:
Wait.. I thought access to MetadataClient was under a lock. But it isn't so here. Won't it affect parallel operations?